src/modules/vehicle/drivetrain.h

```diff
@@ -4,13 +4,40 @@
 namespace MM2
 {
     // Forward declarations
-
+    class vehDriveTrain;
```

Comment:
The class name should be `vehDrivetrain` (no capital T on train)

Reply:
I will change that, thanks for notifying me!